src/mapml/layers/MapLayer.js

```diff
@@ -1120,15 +1146,15 @@ export var MapMLLayer = L.Layer.extend({
       let extent = this._extent;
       if(!extent) return FALLBACK_PROJECTION;
       switch (extent.tagName.toUpperCase()) {
-        case "EXTENT":
+        case "MAP-EXTENT":
           if(extent.hasAttribute('units'))
             return extent.getAttribute('units').toUpperCase();
           break;
         case "INPUT":
```

Comment:
I believe this has to be updated as well since the test files use map-input rather than input now.